OpenConsole.sln

```diff
@@ -2798,6 +2800,32 @@ Global
 		{37C995E0-2349-4154-8E77-4A52C0C7F46D}.Release|x64.Build.0 = Release|x64
 		{37C995E0-2349-4154-8E77-4A52C0C7F46D}.Release|x86.ActiveCfg = Release|Win32
 		{37C995E0-2349-4154-8E77-4A52C0C7F46D}.Release|x86.Build.0 = Release|Win32
+		{2C836962-9543-4CE5-B834-D28E1F124B66}.AuditMode|Any CPU.ActiveCfg = AuditMode|Win32
+		{2C836962-9543-4CE5-B834-D28E1F124B66}.AuditMode|ARM.ActiveCfg = AuditMode|Win32

```

Comment:
We must include a _configuration mapping_ for every platform|config combo in the solution.

Since these do not have a `.0` entry (instead of `.ActiveCfg`), they will not be considered for build.

This and the above are in the correct state :)